README.md

```diff
@@ -41,6 +41,23 @@ This Discord bot allows you to run commands in a Discord channel like this:
 ~command
 ```
 
+### Configuring Commands
+
+By default, all commands are locked down and only can be run by users with ids
+in the AUTHORIZED_USERS environmental variable. 
+
+If you would like a command to be runnable by anyone (not just those
+defined in AUTHORIZED USERS), you need to add it to the NO_AUTH_COMMANDS constant.
+
+**lib.rs**
+```rust
+// These commands do not require a user
+// to be in the AUTHORIZED_USERS env variable
+const NO_AUTH_COMMANDS: &'static [&'static str] = &["ping", "multiply"];
```

Comment:
It doesn't change anything in practice, but you can omit `'static` in `const`s and `static`s.

Reply:
Done!